README.md

```diff
@@ -50,24 +50,21 @@ To run all tests, use the following command:
 
     nosetests tests
 
-## Deployment
+## Releasing an ok-client version
 
-To deploy a new version of ok-client, do the following:
+First make sure that
 
-1. Change the version number in `client/__init__.py`.
-2. Make sure your virtualenv is activated. Also make sure that your `~/.pypirc`
-   contains okpy's Pypi credentials.
-3. From the base of the repo, make sure your virtualenv is activated and run
+* Your virtualenv is activated and you are on the master branch.
+* Your `~/.pypirc` contains okpy's PyPI credentials.
```

Comment:
The tests only take a few seconds, so I'll just add that to the script's sanity checks.